### ISSUE TITLE
Fix Pick Order card design

### ISSUE THREAD

**Issue by marykatefain** [HarvestHub/GardenHub#84]
The divider on the cards gets weird with tall cards:

![screenshot from 2018-01-11 19 01 11](https://user-images.githubusercontent.com/17955536/34853482-efe177fc-f701-11e7-80cf-b32c0f82c90f.png)

**Comment by alexgleason** [comment]
I don't think this will be simple to change, short of removing the bar entirely.

**Comment by marykatefain** [comment]
Oh, alright well it's not priority for now. It's a v minor thing
